code/game/objects/items/RPD.dm

```diff
@@ -479,7 +479,8 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 					if (target_differences & S.connections)
 						to_chat(user, span_warning("\The [src]'s screen flashes a warning: Can't configure a pipe in a currently connected direction."))
 						return
-					var/new_dir = (S.GetInitDirections() & ~target_differences) | target_differences
+					var/old_dir = S.GetInitDirections()
```

Comment:
While we're here, why create old_dir in the first place? There's already a var in this proc which should have the exact same value right?

Reply:
The idea is that the pipe's configuration can change during `do_after()`, so we're checking to see that we're not about to break any newly made connections. `			// Double check to make sure that nothing has changed. If anything we were about to change is now connected, abort` could probably be a little clearer about this